website/docs/getting-started/setup/installation-guides/aws-ecs.md

```diff
@@ -7,180 +7,195 @@ sidebar_position: 5
 
 ## Prerequisites
 
-### 1: Register With Amazon Web Services
+### 1: Register with Amazon Web Services
```

Comment:
```suggestion
### 1. Register with Amazon Web Services
```
nitpick. the colons look odd. keep periods. if we are making some changes there, we might as well fix it one go.